test/test_filters.rb

```diff
@@ -940,12 +940,12 @@ def to_liquid
 
         results = @filter.where(hash, "featured", "true")
         assert_equal 2, results.length
-        assert_equal 9.2, results[0]["rating"]
-        assert_equal 4.7, results[1]["rating"]
+        assert_in_delta(9.2, results[0]["rating"])
```

Comment:
Can we revert these changes?
I feel that `assert_equal` is more intuitive than `assert_in_delta` ...

Reply:
> Use assert_in_delta if comparing floats. Assertion passes if the expected value is within the delta of actual value.

This is intended to compare floats,  it's what we're doing here, something you have to get used to maybe, but once you know it, it makes the test as intuitive as relying on a generic equal.